docs/FAQ.adoc

```diff
@@ -0,0 +1,99 @@
+
+link:{project}/issues/73[Later releases] will allow other users to query Korrel8r within the limits of their access.
+
+== Errors installing on Openshift 4.15
+
+
+Openshift 4.15 enforces additional security restrictions that did not exist up to Openshift 4.14.
+Installing the operator bundle gives security policy errors.
+There are two work-arounds:
+
+. Deploy Korrel8r in the `default` namespace instead of creating a new `korrel8r` namespace.
++
+[source,bash]
+----
+operator-sdk -n default run bundle quay.io/korrel8r/operator-bundle:latest
+----
+
+. Apply labels to the `korrel8r` namespace before installing:
++
+[source,bash]
+----
+oc label ns/korrel8r pod-security.kubernetes.io/enforce=privileged --overwrite
+oc label ns/korrel8r pod-security.kubernetes.io/warn=privileged --overwrite
+----
+
+This will be fixed in future, see also https://issues.redhat.com/browse/OU-304.
+
+== Openshift Console Alert does not show "see related logs"
+
+TIP: Make sure you have https://korrel8r.github.io/korrel8r/#installing-korrel8r-ocp[installed Korrel8r]
+
+The "see related logs" link will not appear for alerts that do not have related logs.
+For example, the `UpdateAvailable` alert for cluster software updates is not connected to any specific Pod,
+so it has no related logs.
+
+One way to generate an alert with related logs is to create a broken deployment in a system namespace like `default` as follows:
+
+[source,bash]
+----
+kubectl apply -f - << EOF
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: bad-deployment
+  namespace: default <1>
+spec:
+  selector:
+    matchLabels:
+      app: bad-deployment
+  template:
+    metadata:
+      labels:
+        app: bad-deployment
+    spec:
+      containers: <2>
+      - name: bad-deployment
+      	image: quay.io/openshift-logging/vector:5.8
+----
+
+<1> The deployment must be in a system namespace like `default` to cause an alert.
+<2> The container will generate some logs and then exit with an error because there is no configuration file.
+
+Alerts appear in the `Pending` state after a minute or so, and transition to `Firing` after 15 minutes.
+Open `Observe > Alerting` and click `clear all filters` to see `Pending` alerts.
+Look for alerts naemd `KubeContainerWaiting`, `KubePodCrashLooping` or `KubePodNotReady`.
+Click an alert to see the `Details` page. You should see a `related logs` link.
+
+==== Openshift Logging collector logs are not available.
+
+The Openshift Loggging log collector does not collect its own logs!
+
+This would create a risk of a circular condition where collecting its own logs causes the collector to log something, which the collector then collects, which causes it to log something and so on.
+This creates an explosion of self-induced log messages.
+
+To avoid this risk the log collector does not collect logs from Pods that are part of the logging system.
+You can still see the logs using `kubectl logs` directly.
```

Comment:
```suggestion
Collecting its own logs would cause a circular condition, where the collector collects a log, which creates a log, which the collector then collects again, and so on. This creates an explosion of self-induced log messages.

To avoid this risk, the log collector does not collect logs from pods that are part of the logging system.
You can still view logging for Red{nbsp}Hat OpenShift logs by using the `oc logs` command directly.
```